pycqed/measurement/multi_qubit_module.py

```diff
@@ -241,11 +247,6 @@ def get_multiplexed_readout_detector_functions(qubits, nr_averages=2 ** 10,
                 integration_length=max_int_len[uhf], nr_shots=nr_shots,
                 get_values_function_kwargs=classif_det_get_values_kws,
                 result_logging_mode='raw', **kw),
-            'int_avg_classif_corr_det': det.UHFQC_integration_average_classifier_det(
-                UHFQC=uhf_instances[uhf], AWG=AWG, channels=channels[uhf],
-                integration_length=max_int_len[uhf], nr_shots=nr_shots,
-                get_values_function_kwargs=classif_det_get_values_kws,
-                correlations=correlations[uhf], result_logging_mode='raw', **kw),
```

Comment:
Why was this removed?

Reply:
Because it's not needed. The classifier detector has 4 different options currently on Proj_XLD: averaged, correlated, thresholded, classified. These options are specified by the user in "get_values_function_kwargs." This seemed cleaner than to make 4 different instances of that detector function, but is probably not be the best solution.

Reply:
Ok, thats good, I was just curious.